build.gradle

```diff
@@ -34,7 +34,7 @@ subprojects {
     apply plugin: 'findbugs'
 
     group = 'io.minio'
-    version = '3.0.4'
+    version = '3.0.5'
```

Comment:
Basically this PR makes devel build (3.0.5-DEV) to differentiate current release 3.0.4, else devel build will conflict with current release.